barretenberg/cpp/src/barretenberg/eccvm/eccvm_verifier.cpp

```diff
@@ -21,6 +21,12 @@ bool ECCVMVerifier::verify_proof(const HonkProof& proof)
 
     for (auto [comm, label] : zip_view(commitments.get_wires(), commitment_labels.get_wires())) {
         comm = transcript->template receive_from_prover<Commitment>(label);
+        // TODO(https://github.com/AztecProtocol/barretenberg/issues/1017): This is a hack to ensure zero commitments
+        // are still on curve as the transcript doesn't currently support a point at infinity representation for
+        // cycle_group
+        if (!comm.on_curve()) {
```

Comment:
Oof this feels dangerous, at least for debugging. Can you make a separate issue "remove point at infinity hack in ECCVM" that we will set as high priority to ensure we handle this even if we don't properly handle point at infinity across all cases?

Reply:
removing, fixed by the point at infinity PR